libfreerdp/core/gateway/ntlm.c

```diff
@@ -223,21 +225,33 @@ BOOL ntlm_authenticate(rdpNtlm* ntlm)
 	}
 
 	status = ntlm->table->InitializeSecurityContext(&ntlm->credentials,
-			(ntlm->haveContext) ? &ntlm->context : NULL,
-			(ntlm->ServicePrincipalName) ? ntlm->ServicePrincipalName : NULL,
-			ntlm->fContextReq, 0, SECURITY_NATIVE_DREP,
-			(ntlm->haveInputBuffer) ? &ntlm->inputBufferDesc : NULL,
-			0, &ntlm->context, &ntlm->outputBufferDesc,
-			&ntlm->pfContextAttr, &ntlm->expiration);
-
+							(ntlm->haveContext) ? &ntlm->context : NULL,
+							(ntlm->ServicePrincipalName) ? ntlm->ServicePrincipalName : NULL,
+							ntlm->fContextReq, 0, SECURITY_NATIVE_DREP,
+							(ntlm->haveInputBuffer) ? &ntlm->inputBufferDesc : NULL,
+							0, &ntlm->context, &ntlm->outputBufferDesc,
+							&ntlm->pfContextAttr, &ntlm->expiration);
+
```

Comment:
I know there are no official guidelines on formatting for FreeRDP (expect read the different formatted code ;() but looking at other parts where indentation is used for long functions I'd say the extra intended part should start at the beginning of the function name (not on the brackets) or is intended with one (or two in extreme cases) extra tabs.